test-three/amy-menu/src/App.css

```diff
@@ -0,0 +1,24 @@
+.App {
```

Comment:
Have you used any CSS in JS frameworks like styled components, emotionjs, glamor etc? One need thing is that it cal allow you to use props to change css.

Reply:
Yes, I have experience using styled components.